TINKERPOP-2824 Properties on response elements (#1843)
apache/tinkerpop: docs/src/reference/gremlin-applications.asciidoc

With the type of request and detachment model in mind, it is now possible to discuss how best to consider element
properties in relation to them all in concert.

By default, Gremlin Server sample configurations return all properties.

To manage properties for each request you can use <<configuration-steps-with,with()>> configuration option 
`materializeProperties`

[source,groovy]
----
g.with('materializeProperties', 'tokens').V()
----

> The `tokens` value for the `materializeProperties` means that need to return only `id` and `label`. Another option

Reviewer: ```suggestion
The `tokens` value for the `materializeProperties` means that only `id` and `label` should be returned. Another option
```

> `all` used to indicate that all properties should be returned and is the default value.

Reviewer: I think this needs to be:

> Another option, `all`, _can be_ used [...]
